app/views/profiles/_edit.haml

```diff
@@ -9,10 +9,22 @@
   = render 'profiles/edit_public', :profile => profile, :aspect => aspect, :person => person
 
   %hr
+  :javascript
```

Comment:
What is a proper place for javascript for this page in the actual diaspora codebase? I tried app/assets/javascript/app/pages/profile.js, but it seems that it doesn't get loaded on profile/edit page.